app/src/main/java/fr/iut_rodez/pathpilot_android_client/home/Home.java

```diff
@@ -0,0 +1,75 @@
+package fr.iut_rodez.pathpilot_android_client.home;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ImageView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.viewpager2.widget.ViewPager2;
+
+import com.google.android.material.tabs.TabLayout;
+import com.google.android.material.tabs.TabLayoutMediator;
+
+import fr.iut_rodez.pathpilot_android_client.R;
+import fr.iut_rodez.pathpilot_android_client.home.clients.FragmentClients;
+import fr.iut_rodez.pathpilot_android_client.login.LoginService;
+import fr.iut_rodez.pathpilot_android_client.login.TokenJWT;
+
+public class Home extends AppCompatActivity {
```

Comment:
class Javadoc comments to add

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994